spec/rmt/wizard_scc_page_spec.rb

```diff
@@ -33,6 +33,7 @@
 
       expect(Yast::UI).to receive(:ChangeWidget).with(Id(:scc_username), :Value, config['scc']['username'])
       expect(Yast::UI).to receive(:ChangeWidget).with(Id(:scc_password), :Value, config['scc']['password'])
+      # expect(Yast::Wizard::Content).to receive(:PushButton).with(Id(:skip), Opt(:key_F11), Yast::Label.SkipButton)
```

Comment:
we usually test it with setting content and with benefit of nested find of Yast::Term like
https://github.com/yast/yast-storage-ng/blob/master/test/y2storage/dialogs/proposal_test.rb#L240 ( some helpers involved, but I hope you get idea ). Ask if you have any questions.

Reply:
Thanks @jreidinger it worked out great :+1: